pkg/console/starter/starter.go

```diff
@@ -324,19 +325,20 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 	)
 
 	// NOTE: be sure to uncomment the .Run() below if using this
-	//staleConditionsController := staleconditions.NewRemoveStaleConditions(
-	//	[]string{
-	//		// If a condition is removed, we need to add it here for at least
-	//		// one release to ensure the operator does not permanently wedge.
-	//		// Please do something like the following:
-	//		//
-	//		// example: in 4.x.x we removed FooDegraded condition and can remove
-	//		// this in 4.x+1:
-	//		// "FooDegraded",
-	//	},
-	//	operatorClient,
-	//	controllerContext.EventRecorder,
-	//)
+	staleConditionsController := staleconditions.NewRemoveStaleConditionsController(
+		[]string{
+			// If a condition is removed, we need to add it here for at least
+			// one release to ensure the operator does not permanently wedge.
+			// Please do something like the following:
+			//
+			// example: in 4.x.x we removed FooDegraded condition and can remove
+			// this in 4.x+1:
+			// "FooDegraded",
```

Comment:
Can you update this comment to explain that the removed condition was misspelled and what release that can be removed in?